### PR TITLE
Chore: Add new deps

### PR DESCRIPTION
**Pull request by tmcconechy**
**Explain the details for making this change. What existing problem does the pull request solve?**
Updates a few deps. Mainly a test to confirm https://github.com/percy/cli/issues/1194#event-8694897940 by running a build.

NOTE: Percy Support might be looking at this. Hold until WIP is removed.

**Related github/jira issue (required)**:
https://github.com/percy/cli/issues/1194

**Steps necessary to review your pull request (required)**:
- checks pass
- `npm i` is clean of errors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,42 +53,42 @@
     "@axe-core/puppeteer": "^4.6.0",
     "@babel/eslint-parser": "^7.19.1",
     "@babel/preset-env": "^7.20.2",
-    "@percy/cli": "1.18.0",
+    "@percy/cli": "1.20.3",
     "@percy/puppeteer": "2.0.2",
     "@types/jest": "29.4.0",
     "@types/jest-axe": "3.5.5",
-    "@types/node": "^18.14.4",
+    "@types/node": "^18.14.6",
```
Comment (EdwardCoyle) [inline package.json:60]: Wondering if this should match the recommended Node version in `.nvmrc`, which is lower: https://github.com/infor-design/enterprise-wc/blob/new-deps/.nvmrc#L1

**Review by EdwardCoyle — APPROVED** [review]
Everything seems fine, but a question below about one item:

File: package.json (hunk 1 above)
Comment (tmcconechy) [reply package.json:60]: Not 100% sure but the latest is 18.15.0 there https://nodejs.org/en/download/ will just make them match for now. Having some wierd test issues. Might have to fix this later

Comment (tmcconechy) [reply package.json:60]: Yeah 18.14.6 doesnt exist so i guess they cant match (different packages at the end of the day)

**Pull request closed without merging**